cogs5e/models/character.py

```diff
@@ -97,9 +100,14 @@ def from_dict(cls, d):
         return inst
 
     @classmethod
-    async def from_ctx(cls, ctx):
+    async def from_ctx(cls, ctx, ignoreGuild: bool = False):
         owner_id = str(ctx.author.id)
-        active_character = await ctx.bot.mdb.characters.find_one({"owner": owner_id, "active": True})
+        active_character = None
+        if ctx.guild and not ignoreGuild:
```

Comment:
`if ctx.guild` should be `if ctx.guild is not None` for readability and in case the truthiness value of guilds ever changes.

Reply:
```suggestion
        if ctx.guild is not None and not ignoreGuild:
```